modules/codenvy/templates/cleanUpWorkspaceStorage.sh.erb

```diff
@@ -1,6 +1,16 @@
 #!/bin/sh
 
 DST_FOLDER=${1}
-CLEANUP_REGEX='^(/opt/codenvy-data/fs/([0-9a-f]{2}/){3}workspace[0-9a-z]+)$'
+CLEANUP_REGEX='^(.*/codenvy-data/fs/([0-9a-f]{2}/){3}workspace[0-9a-z]+)$'
 
-[[ ${DST_FOLDER} =~ ${CLEANUP_REGEX} ]] && sudo rm -rf "${DST_FOLDER}"
+hasSudoCommand() {
+  hash sudo 2>/dev/null && return 0 || return 1
+}
+
+if echo "${DST_FOLDER}" | grep -q -E "${CLEANUP_REGEX}"; then
```

Comment:
that script written in a way to be compatible with old codenvy where we have sudo. So for now we want to keep it like that and once codenvy will be only in docker we will clean this up.